Modules/PSGraph.md

```diff
@@ -0,0 +1,21 @@
+# Name: PSGraph
+
+## Description:
+PSGraph is a module that is implemented as a DSL for the GraphViz engine. This allows you to generate graphs of information with Nodes and Edges dynamically from your data.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| [GraphViz](http://graphviz.org/) | Kevin Marquette (<a href="https://kevinmarquette.github.io/blog/?utm_source=PowerShellModulesCentral&utm_medium=markdown" target="_blank">b</a> \| <a href="https://twitter.com/kmarquette" target="_blank">t</a>) | [Github](https://github.com/KevinMarquette/PSGraph) \| [official project site](https://kevinmarquette.github.io/) | Jan 2017 | Graph, DSL, Visualization |
+
+____
+### Shared blog posts:
+Kevin Marquette (<a href="https://kevinmarquette.github.io/blog/?utm_source=PowerShellModulesCentral&utm_medium=markdown" target="_blank">b</a> \| <a href="https://twitter.com/kmarquette" target="_blank">t</a>)
+- [Powershell: PSGraph, A graph module built on GraphViz](https://kevinmarquette.github.io/2017-01-30-Powershell-PSGraph//?utm_source=PowerShellModuleCentral&utm_medium=markdown) @ 2017-01-30
```

Comment:
At the end of this line you need to have 2 spaces.
Otherwise the PSGraph Docs will appear on the same line. (even with the carriage return)

Here is the actual preview:

![image](https://user-images.githubusercontent.com/19521315/30786827-e14a76a0-a173-11e7-949a-25f493d1cdbd.png)

Reply:
I pulled that from this section and moved it up to the official lings